node/sc/bridgepool/sorted_map_list.go

```diff
@@ -100,6 +100,23 @@ func (m *ItemSortedMap) Put(event itemWithNonce) {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 
+	m.put(event)
+}
+
+// PutWithLimit inserts a new item into the map with size limit of the map.
+func (m *ItemSortedMap) PutWithLimit(event itemWithNonce, limit int) {
```

Comment:
How about simply setting the `limit` in the ItemSortedMap itself?

Reply:
This `ItemSortedMap` is used for this PR and BridgeTxPool. I don't want to modify the construct(`NewItemSortedMap`). What do you think?

Reply:
If we need to limit the size of `ItemSortedMap` in general, I think it is better to set the limit in `ItemSortedMap`.

Reply:
I accepted you guys opinion. I appreciate your good opinion. = )
@jeonghonah @ehnuje @nogang